client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -33,6 +33,9 @@ export function Gutenboard() {
 		case Step.DesignSelection:
 			prev = Step.IntentGathering;
 			break;
+		case Step.PageSelection:
+			prev = Step.DesignSelection;
+			break;
```

Comment:
Ah, good point, forget that when we were adding the Page Selector route (#39210)! :sweat_smile:

Reply:
Yes totally missed that one.